include/boost/geometry/io/wkt/write.hpp

```diff
@@ -62,22 +62,63 @@ namespace boost { namespace geometry
 namespace detail { namespace wkt
 {
 
+template <typename Output>
+struct output_formatter
+{
+    template <typename T>
+    static void add(Output & out, T const& v)
+    {
+        out << v;
+    }
+};
+
+template <>
+struct output_formatter<std::string>
+{
+    template <typename T>
+    static void add(std::string & out, T const& val)
+    {
+        std::stringstream ss;
+        ss << val;
+        out += ss.str();
+    }
+
+    static void add(std::string & out, char c)
+    {
+        out += c;
+    }
+
+    static void add(std::string & out, const char * s)
```

Comment:
Nitpick: `const char *` -> `char const*`

Reply:
Does this rule also apply for pointers, or only references? I have a feeling that this can be confused with `char * const` by a beginner.

Reply:
Also of you change `std::string` to `std::basic_string` then it becomes `CharT const*` or `const CharT *`.

Reply:
```
T const&
T const*
```

is clearest yet most symmetric syntax possible, also for pointers as even if one writes `const T*`, correct reading is from right to left anyway. I'd actually expect beginners having problems with `const T*` ;)

This was discussed quite some time ago and documented here https://github.com/boostorg/geometry/wiki/Guidelines-for-Developers